server/src/main/scala/za/co/absa/atum/server/api/service/BaseService.scala

```diff
@@ -17,25 +17,20 @@
 package za.co.absa.atum.server.api.service
 
 import za.co.absa.atum.server.api.exception.{DatabaseError, ServiceError}
-import za.co.absa.fadb.exceptions.StatusException
+import za.co.absa.db.fadb.exceptions.StatusException
 import zio._
 
 trait BaseService {
 
-  def repositoryCall[R](
-    repositoryCall: IO[DatabaseError, R],
-    operationName: String
-  ): IO[ServiceError, R] = {
+  def repositoryCall[R](repositoryCall: IO[DatabaseError, R], operationName: String): IO[ServiceError, R] = {
     repositoryCall
       .mapError { case DatabaseError(message) =>
         ServiceError(s"Failed to perform '$operationName': $message")
       }
   }
 
-  def repositoryCallWithStatus[R](
-    repositoryCall: IO[DatabaseError, Either[StatusException, R]],
-    operationName: String
-  ): IO[ServiceError, Either[StatusException, R]] = {
+  def repositoryCallWithStatus[R](repositoryCall: IO[DatabaseError, Either[StatusException, R]], operationName: String
```

Comment:
Not used since we decided to handle StatusException(s) on repository level. Now the question is whether we will always propagate StatusExceptions to upper layers (Controller) in cases when we really need to (for instance when we will need to decide on resulting status code of given REST call based on the status of the database call) or we will create a hierarchy of ServiceError(s) indicating what happened on the repository level and propagate that information this way using ZIO's error channel.